source/reference/options.txt

```diff
@@ -160,13 +160,15 @@ Built-In Variables
 ------------------
 
 - ``{{role}}``: automatically replaced by a link to the option or
-  setting (e.g. :option:`mongod.--dbpath`).
+  setting (e.g. :manual:`dbpath </reference/program/mongod/#std-option-mongod.--dbpath>`).
 
 - ``{{program}}``: automatically replaced by the a link to the option
-  or setting's specified ``program`` (e.g. :binary:`~bin.mongod`).
+  or setting's specified ``program`` (e.g. :manual:`mongod </reference/program/mongod/`).
   
 - ``{{command}}``: automatically replaced by a link to the option or
-  setting's specified ``command`` (e.g. :toolcommand:`record`).
+  setting's specified ``command``.
+
+.. (e.g. :toolcommand:`record`).
```

Comment:
Just making sure you meant to comment out the e.g. here?

Reply:
yes I did!